RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -215,4 +219,22 @@ void auto_LegacyOfLoathingDailies()
 	{
 		use(1, $item[replica Smith\'s Tome]); // get items
 	}
+
+	if(item_amount($item[Replica 2002 Mr. Store Catalog]) > 0)
```

Comment:
also check prop `_2002MrStoreCreditsCollected` is false

Reply:
speaking of which, would be good to update conditions above this as well if you are up for it.

`_replicaSmithsTomeUsed` and `_replicaResolutionLibramUsed` would be the prefs to check